Common/tests/Submitter/SubmitterTests.cs

```diff
@@ -310,6 +312,32 @@ public async Task CreateSessionShouldSucceed()
                     result.SessionIds.Single());
   }
 
+  [Test]
+  public async Task CreateSessionWithoutPartition()
+  {
+    var defaultTaskOptions = new TaskOptions
+                             {
+                               MaxDuration = Duration.FromTimeSpan(TimeSpan.FromSeconds(2)),
+                               MaxRetries  = 2,
+                               Priority    = 1,
+                               PartitionId = "invalid",
+                             };
+
+    /*
+    // TODO: Once partitions are fully integrated, creating a session without partition should fail
```

Comment:
One can avoid the commented code by adding the attribute [Ignore("reason to ignore the test")]  to the test